sdb/commands/linux/internal/slub_helpers.py

```diff
@@ -0,0 +1,108 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+from typing import Iterable
+
+import drgn
+from drgn.helpers.linux.list import list_for_each_entry
+
+
+def is_root_cache(obj: drgn.Object) -> bool:
+    return obj.memcg_params.root_cache.value_() == 0x0
```

Comment:
when do we need to use `.value()`?  I think the drgn.Object implements `==` and you might be able to do `obj.memcg_params.root_cache == 0x0`?

Reply:
Per our conversation, this is unfortunately not possible with the `drgn` API currently:
```
>>> a.memcg_params.root_cache
(struct kmem_cache *)0x0
>>> a.memcg_params.root_cache + 1
(struct kmem_cache *)0x2158
>>> a.memcg_params.root_cache == 0x0
Traceback (most recent call last):
  File "<console>", line 1, in <module>
TypeError: invalid operands to comparison ('struct kmem_cache *' and 'int')
```